fvm/fvm_test.go

```diff
@@ -1848,7 +1848,8 @@ func TestSigningWithTags(t *testing.T) {
 
 				err = vm.Run(ctx, tx, view, programs)
 				require.NoError(t, err)
-				require.NoError(t, tx.Err)
+				require.Error(t, tx.Err)
+				require.Equal(t, errors.ErrCodeInvalidProposalSignatureError, tx.Err.(errors.Error).Code())
```

Comment:
I think we can use `require.ErrorIs` here

Reply:
ErrrorIs and ErrorAs didn't work for me here. Not sure whats going on. I would probably need to define Is ad As on the error types.

I instead switched to IsType.